### PR TITLE
Add qbft_getRequestTimeoutSeconds RPC Method

### PR DESCRIPTION
**Pull request by 7suyash7**
Fixes: #7032 

Adds a new RPC method `qbft_getRequestTimeoutSeconds` to allow querying the current request timeout seconds value for QBFT nodes.

Output:
```
curl -X POST --data '{"jsonrpc":"2.0","method":"qbft_getRequestTimeoutSeconds","params":[],"id":1}' http://localhost:8545                               
{"jsonrpc":"2.0","id":1,"result":10}
```

### CODE REVIEW

**Comment by 7suyash7** [comment]
@macfarla If this gets merged, it would make sense to add the same method for IBFT nodes as well right? That could be in another PR

**Review by jframe — DISMISSED** [review]
LGTM

File: consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/jsonrpc/methods/QbftGetRequestTimeoutSecondsTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ExtendWith(MockitoExtension.class)
+public class QbftGetRequestTimeoutSecondsTest {
+
+  private static final Logger LOG = LoggerFactory.getLogger(QbftGetRequestTimeoutSecondsTest.class);
```
Comment (jframe) [inline consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/jsonrpc/methods/QbftGetRequestTimeoutSecondsTest.java:37]: This logger isn't being used and is causing the build to fail

**Comment by 7suyash7** [comment]
@jframe Hi, I just removed it, can you please re-run

**Comment by macfarla** [comment]
@7suyash7 there are some spotless errors to fix

**Comment by 7suyash7** [comment]
> @7suyash7 there are some spotless errors to fix

Can you please re-run

**Review by jframe — APPROVED** [review]

**Pull request merged**